### PR TITLE
Update applications and other domain texts

### LINKED ISSUE #4601: Update applications domain texts
### Description

Update all the texts under applications domain

### Acceptance Criteria

- Review all the texts under [applications domain](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=41%3A52388&t=prTQCIyJ1X49773R-4) and update the texts according to proposal 
- [Remove applications](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=129%3A79866&t=prTQCIyJ1X49773R-4)
- [Disconnect application](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=195%3A77813&t=prTQCIyJ1X49773R-4)
- [Bridge application](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=195%3A77815&t=prTQCIyJ1X49773R-4)
- [Application connect flow](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=41%3A52384&t=prTQCIyJ1X49773R-4)
- [Sign transaction flow](https://www.figma.com/file/KcrDpvWEKQhdGwNd4CZ5NY/Desktop-Prototype?node-id=41%3A52386&t=prTQCIyJ1X49773R-4)

### PR DESCRIPTION
**Pull request by ManuGowda**
### What was the problem?

This PR resolves #4601 

### How was it solved?

:memo: Update grammer and sentences

### How was it tested?

- Updated tests
- Run and review the application

### CODE REVIEW

File: src/modules/account/components/BackupRecoveryPhraseFlow/BackupRecoveryPhraseFlow.js [hunk 1]
```diff
@@ -36,10 +36,10 @@ const BackupRecoveryPhraseFlow = ({ history }) => {
           <EnterPasswordForm
             onEnterPasswordSuccess={onEnterPasswordSuccess}
           />
-          <SavePassphrase passphrase={passphrase} />
+          <SavePassphrase passphrase={passphrase} title='Back up your secret recovery phrase' />
```
Comment (soroushm) [inline src/modules/account/components/BackupRecoveryPhraseFlow/BackupRecoveryPhraseFlow.js]: please translate it here
```suggestion
          <SavePassphrase passphrase={passphrase} title={t('Back up your secret recovery phrase')} />
```

File: src/modules/auth/components/EnterPasswordForm/EnterPasswordForm.js [hunk 2]
```diff
@@ -53,9 +53,9 @@ const EnterPasswordForm = ({ onEnterPasswordSuccess, title, encryptedAccount, is
   return (
     <Box className={styles.container}>
       <BoxContent className={styles.content}>
-        <h1>{t('Enter your password')}</h1>
+        <h1>{t('Enter your account password')}</h1>
         <p className={styles.subheader}>
-          {t(title || 'Please provide your device password to backup the recovery phrase.')}
+          {t(title || 'Please enter your account password to backup the secret recovery phrase.')}
```
Comment (soroushm) [inline src/modules/auth/components/EnterPasswordForm/EnterPasswordForm.js]: i18n scan cant scan it in this way
```suggestion
          {title || t('Please enter your account password to backup the secret recovery phrase.')}
```

File: src/modules/account/components/AddAccountForm/AddAccountForm.js [hunk 3]
```diff
@@ -75,7 +75,7 @@ const AddAccountFormContainer = ({
               </fieldset>
             )}
             <fieldset>
-              <label>{t('Secret recovery phrase')}</label>
+              <label>{t('Secret recovery phrase (12 -24 paraphase supported)')}</label>
```
Comment (soroushm) [inline src/modules/account/components/AddAccountForm/AddAccountForm.js]: typo
```suggestion
              <label>{t('Secret recovery phrase (12-24 paraphrase supported)')}</label>
```

File: src/modules/account/components/EditAccount/EditAccount.js [hunk 4]
```diff
@@ -17,8 +17,8 @@ const EditAccount = ({ history }) => {
       <EditAccountForm onBack={history.goBack} />
       <SetPasswordSuccess
         encryptedPhrase={currentAccount}
-        headerText={t('Edit account name successful')}
-        contentText={t('You can now download encrypted secret recovery phrase to this effect.')}
+        headerText={t('Edit account name')}
+        contentText={t('Successfully edited the account name, now you can download the encrypted secret recovery phrase to this effect.')}
```
Comment (soroushm) [inline src/modules/account/components/EditAccount/EditAccount.js]: its a bit too long could we make it shorter?
```suggestion
        contentText={t('Successfully edited, now you can download the encrypted secret recovery phrase to this effect.')}
```

Comment (soroushm) [inline src/modules/account/components/EditAccount/EditAccount.js]: its a bit too long could we make it shorter?
```suggestion
        contentText={t('Successfully edited, now you can download the encrypted secret recovery phrase to this effect.')}
```

File: src/modules/account/components/AddAccountForm/AddAccountForm.js (hunk 3 above)
Comment (soroushm) [reply src/modules/account/components/AddAccountForm/AddAccountForm.js]: shouldnt be called phrase like the other places?

File: src/modules/account/components/AddAccountOptions/AddAccountOptions.js [hunk 5]
```diff
@@ -38,7 +38,7 @@ const AddAccountOptions = ({ history }) => {
                 onClick={() => history.push(routes.addAccountBySecretRecovery.path)}
               />
               <AddAccountOptionButton
-                text="Restore from file"
+                text="Restore from backup"
```
Comment (soroushm) [inline src/modules/account/components/AddAccountOptions/AddAccountOptions.js]:
```suggestion
                text={t("Restore from backup")}
```

File: src/modules/auth/components/SavePassphrase/SavePassphrase.js [hunk 6]
```diff
@@ -6,11 +6,11 @@ import PassphraseBackup from '@auth/components/passphraseBackup';
 import registerStyles from '../Signup/register.css';
 
 const SavePassphrase = ({
-  t, passphrase, prevStep, nextStep,
+  t, passphrase, prevStep, nextStep, title,
 }) => (
   <>
     <div className={registerStyles.titleHolder}>
-      <h1>{t('Save your secret recovery phrase')}</h1>
+      <h1>{t(title ?? 'Save your secret recovery phrase')}</h1>
```
Comment (soroushm) [inline src/modules/auth/components/SavePassphrase/SavePassphrase.js]:
```suggestion
      <h1>{title || t( 'Save your secret recovery phrase')}</h1>
```

File: src/modules/auth/components/SetPasswordSuccess/SetPasswordSuccess.js [hunk 7]
```diff
@@ -43,7 +43,7 @@ function SetPasswordSuccess({
 SetPasswordSuccess.defaultProps = {
   onClose: () => null,
   encryptedPhrase: {
-    error: 'no encrypted backup found',
+    error: 'no encrypted back up found',
```
Comment (soroushm) [inline src/modules/auth/components/SetPasswordSuccess/SetPasswordSuccess.js]:
```suggestion
    error: 'no encrypted backup found',
```

File: src/modules/common/components/DownloadJSON/DownloadJSON.js [hunk 8]
```diff
@@ -32,7 +32,7 @@ function DownloadJSON({ encryptedPhrase, fileName }) {
 
 DownloadJSON.defaultProps = {
   encryptedPhrase: {
-    error: 'no encrypted backup found',
+    error: 'no encrypted back up found',
```
Comment (soroushm) [inline src/modules/common/components/DownloadJSON/DownloadJSON.js]:
```suggestion
    error: 'no encrypted backup found',
```

File: src/modules/message/components/signMessageView/index.js [hunk 9]
```diff
@@ -18,17 +18,23 @@ const signMessageView = ({ account, t, history, signMessage }) => {
...
+        {multiStepPosition === 0 && (
           <BoxHeader>
             <h1>{t('Sign message')}</h1>
           </BoxHeader>
         )}
+        {multiStepPosition === 2 && (
+          <BoxHeader>
+            <h1>{t('Signed message')}</h1>
+          </BoxHeader>
+        )}
```
Comment (soroushm) [inline src/modules/message/components/signMessageView/index.js]: please avoid duplication
```suggestion
        {(multiStepPosition === 0 || multiStepPosition === 2) && (
          <BoxHeader>
            <h1>{t('Sign message')}</h1>
          </BoxHeader>
        )}
       
```

Comment (soroushm) [reply src/modules/message/components/signMessageView/index.js]: better option
```suggestion
const hasHeader = multiStepPosition === 0 || multiStepPosition === 2
        {hasHeader && (
          <BoxHeader>
            <h1>{t('Sign message')}</h1>
          </BoxHeader>
        )}
       
```

File: src/modules/message/components/signMessageView/index.js [hunk 10]
```diff
@@ -18,17 +18,23 @@ const signMessageView = ({ account, t, history, signMessage }) => {
   ] = useCurrentAccount();
 
   const onMultiStepChange = useCallback(({ step: { current } }) => {
+    console.log('....', current);
```
Comment (soroushm) [inline src/modules/message/components/signMessageView/index.js]: remove console.log
```suggestion

```

File: src/utils/paperWallet.js [hunk 11]
```diff
@@ -79,7 +79,7 @@ class PaperWallet {
     this.doc.setFont('gilroy', 'normal', 'normal').setFontSize(14);
     this.doc.text(t('How we recommend to store it.'), 32, 147, textOptions);
     this.doc.text(t('Print it on paper and store it in a safe place'), 76, 194, textOptions);
-    this.doc.text(t('Save it on an encrypted hard drive: USB key or a backup drive'), 76, 238, textOptions);
+    this.doc.text(t('Save it on an encrypted hard drive: USB key or a back up drive'), 76, 238, textOptions);
```
Comment (soroushm) [inline src/utils/paperWallet.js]: same here

**Review by soroushm — CHANGES_REQUESTED** [review]

File: src/modules/message/components/signMessageView/index.js (hunk 9 above)
Comment (ManuGowda) [reply src/modules/message/components/signMessageView/index.js]: The header needs to change based on the state, once the message is signed it shouldn't be called `Sign message`? so it was updated to `Signed message`.

Comment (soroushm) [reply src/modules/message/components/signMessageView/index.js]: we can have the condition only for text
```suggestion
        {multiStepPosition !== 1 && (
          <BoxHeader>
            <h1>{multiStepPosition === 2 ? t('Signed message')  : t('Sign message')}</h1>
          </BoxHeader>
        )}
       
```

**Review by soroushm — APPROVED** [review]

**Pull request merged**